### PR TITLE
Fix #600, adding a few useful commands to the Using Doc.

### LINKED ISSUE #600: Add additional useful commands to the Usage docs
Some additional commands which could be useful for users should be added to the Some useful commands section in the Usage guide 
$ vagrant global-status
$ vagrant plugin list
$ vagrant box list 
$ vagrant provision

### PR DESCRIPTION
**Pull request by Preeticp**
Adding a few commands to the Useful Commands section of the Using Document.

### CODE REVIEW

File: docs/using.adoc [hunk 1]
```diff
@@ -314,3 +314,15 @@ Destroys the Vagrant box permanently.
...
+Displays the active Vagrant environments on your system. Use this to see the status of your Vagrant environment.
+
+`vagrant plugin list`::
+Verifies the vagrant plugins installed on your system. Use this to see a list of the vagrant plugins in your system.
+
+`vagrant box list`::
+Lists the vagrant boxes on your system. You can use this to verify that your box is installed.
+
+`vagrant provision`::
+Use this to provision OpenShift if it fails to start on `Vagrant up`.
```
Comment (brgnepal) [inline docs/using.adoc]: We can also provision other services too. So probably you want to make it general.

File: docs/using.adoc [hunk 2]
```diff
@@ -314,3 +314,15 @@ Destroys the Vagrant box permanently.
 WARNING: Using `vagrant destroy` will destroy any data you stored in
 the Vagrant box. You will not be able to restart this instance and will
 have to create a new one using `vagrant up`.
+
+`vagrant global status`::
+Displays the active Vagrant environments on your system. Use this to see the status of your Vagrant environment.
```
Comment (brgnepal) [inline docs/using.adoc]: Add an option here as `--prune`. You can check its doc here https://www.vagrantup.com/docs/cli/global-status.html

**Comment by thatdocslady** [comment]
LGTM :)

File: docs/using.adoc [hunk 3]
```diff
@@ -314,3 +314,16 @@ Destroys the Vagrant box permanently.
 WARNING: Using `vagrant destroy` will destroy any data you stored in
 the Vagrant box. You will not be able to restart this instance and will
 have to create a new one using `vagrant up`.
+
+`vagrant global status`::
+Displays the state of all active Vagrant environments on your system. Use this to see the status of your Vagrant environment. +
+You can use `vagrant global status --prune` to prune invalid entries, based on stale results.
```
Comment (brgnepal) [inline docs/using.adoc:320]: :+1:   for --prune.   Please updated `global status` to `global-status`.  Doc here https://www.vagrantup.com/docs/cli/global-status.html

**Comment by brgnepal** [comment]
LGTM

**Pull request merged**